python_bindings/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
-##
-# Load Python dependencies, including external pybind11
-##
+cmake_minimum_required(VERSION 3.16...3.23)
+project(Halide_Python)
 
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+#SET(HALIDE_MIN_VERSION "14.0.0")
+find_package(Halide REQUIRED)
```

Comment:
Need to check that exceptions / RTTI are enabled.

Reply:
Also need to disable this `find_package` call when it is included by the parent project

The best way to do this is probably to create a `./cmake/FindHalide.cmake` module that just prints something like `"${PROJECT_NAME}: Using in-tree Halide"`. The `./cmake` folder is already added to `CMAKE_MODULE_PATH`, so it should take precedence over any installed Halide instances in the users' system.

Reply:
> @alexreinking Sorry for the delay, I had to fix some other problems first :D
> 
> I think scitkit build simplified a lot. I pushed a minimum draft, but I don't have the time to fix the missing parts (I also don't understand the stub-thing fully). Maybe you can re-introduce the parts that I removed like "correctness", "tutorials", etc.?
> 
> * I guess it would be nice to move the correctness part into a pytest framework.
> * Maybe you can fix the find_package and the rtti issues. I don't fully understand how to solve it. I saw that we're exporting the RTTI files in the setup.py for DaCe, maybe this helps https://github.com/spcl/dace/blob/master/setup.py

Also for the versioning, description stuff, it makes sense that you fix the missing pieces :)